server/meca/file-generators/__snapshots__/article.test.js.snap

```diff
@@ -0,0 +1,193 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Article XML generator generates expected XML 1`] = `
```

Comment:
Good use of snapshots, made this much easier to review.

Reply:
Yeah. Finally found something snapshots are good for!